src/java/org/jivesoftware/openfire/plugin/rest/AuthFilter.java

```diff
@@ -106,6 +106,7 @@ public void filter(ContainerRequestContext containerRequest) throws IOException
         String auth = containerRequest.getHeaderString("authorization");
 
         if (auth == null) {
+            LOG.warn("REST API request with no Authorization header rejected");
```

Comment:
I'd add some identifying details that the admin can use to track down which client is causing this.